### DISCUSSION TITLE
Override default_args definition using passed params

### DISCUSSION THREAD

**Discussion by a246530** [apache/airflow#21627] [Q&A]
Is there a way to take parameters (via trigger with config) and then have those parameters modify the DAG's default_args or even the tasks default args? 
Everytime I set the 'retries' option on a task to a Jinja templated value such as {{params.retries}} I get an error message the says it cant convert '{{params.retries}}' into an int when trying to import the DAG. 
I have added a user_defined_filter that is converting it to an Int and I know it converts it to an Int because I get an error when I try to join it with a string when printing it out in a python operator task.
This to me indicates that the way airflow compiles the DAG has an order of operations such that before it converts the Jinja template to its value, the retries option tries to pick it up as a string. Whats strange is that I can pass Jinja templates into other fields, such as the command option in the SSH operator without complaint.
Not sure if anyone has any experience with this?

**Comment by jhgoebbert** [comment]
This seems to be related: https://github.com/apache/airflow/discussions/63191
